src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -181,16 +181,52 @@ end
 # Equality
 ######################
 
-function Base.:(==)(tv1::NormalToricVariety, tv2::NormalToricVariety)
-  tv1 === tv2 && return true
-  error("Equality of normal toric varieties is computationally very demanding. More details in the documentation.")
-end
+@doc raw"""
+    (==)(X::NormalToricVariety, Y::NormalToricVariety)
```

Comment:
A while back, we took steps to remove "unnecessary" doc-strings. For instance, we have tons of implementation of `dim`, `base_ring`, `gens`, etc. which only differ by signature. Those doc-strings were removed, so that when you search in the online documentation (or issue ? in the REPL), you get fewer results/have an easier time to find what you are really looking for.

I cannot find this described in the developer documentation (maybe it is there and I just overlooked it), but the case of equality seems to fall into the same category as I see it. @thofma ? My suggestion would be to remove the `@doc`, and instead describe the equality check directly in the corresponding `.md` file.

Reply:
I remember that we removed at lot of redundant docstrings of the form
```
+(x::Bla, y::Bla)

> Returns the sum of x and y
```
(and other functions) for quite a lot signatures. But I can't locate the issue nor the PR (only your tangentially related https://github.com/oscar-system/Oscar.jl/issues/1471).

If think for signatures, where there is a chance of ambiguity what it could mean (like in this case), I think it is fine to keep it for now.

Reply:
Yeah you are right. Thank you for correcting me.

(Added a triage label to my original issue. Maybe this one could be closed, once the decision is described in the style guide...)